packages/@react-spectrum/s2/intl/sl-SI.json

```diff
@@ -2,8 +2,9 @@
   "actionbar.actions": "Dejanja",
   "actionbar.actionsAvailable": "Na voljo so dejanja.",
   "actionbar.clearSelection": "Počisti izbor",
-  "actionbar.selected": "Izbrano: {count}",
-  "actionbar.selectedAll": "Vsi izbrani",
+  "actionbar.selected": "{count, plural, =0 {None selected} drugo {# selected}}",
```

Comment:
```suggestion
  "actionbar.selected": "{count, plural, =0 {None selected} other {# selected}}",
```

Reply:
Thanks @snowystinger I've updated for that and added translation
{count, plural, =0 {Nič ni izbrano} other {# izbrano}}